projects/commudle-admin/src/app/feature-modules/public-community/components/speakers/speakers.component.scss

```diff
@@ -0,0 +1,73 @@
+@import "/projects/commudle-admin/src/assets/styles/variables.scss";
+
+.speakers-container {
+  nb-card {
+    background: transparent;
+    border: none;
+    nb-card-body {
+      .user-card {
+        position: relative;
+        width: 200px;
+        height: 280px;
+        background: $white;
+        transition: all 0.5s ease;
+        box-shadow: 0 3px 6px rgba(000, 000, 0, 0.3);
+        border-radius: 5px;
+        overflow: hidden;
+
+        &:hover {
+          transform: translateY(-5px);
+          box-shadow: 0 5px 15px rgba(000, 000, 0, 0.5);
+        }
+
+        .banner {
+          position: absolute;
+          width: 200px;
```

Comment:
Width should be 100% of the card's width

Reply:
Also, please make sure that the aspect ratio of this banner div is same as that on the user's profile page (where larger banner is displayed)